src/scenarios/shared/runner.py

```diff
@@ -514,6 +528,8 @@ def run(self):
                     getLogger().exception("Failed to get past permission screen, run locally to see if enough next button presses were used.")
                     exit(-1)
 
+            time.sleep(10) # Add delay to ensure app is fully installed and give it some time to settle
```

Comment:
@LoopedBard3 it's not clear to me here, is it sleeping 10 seconds between each run?

Is there a way we can launch the first time, wait 10 seconds, throw that number out. Then do the normal 10 run average?

Reply:
That is what is happening here. The main tests take place from line 534 to 539 and have their own 3 second gap between those tests.

Reply:
Ok sounds good, then. 👍